prepline_general/api/general.py

```diff
@@ -775,7 +769,7 @@ def response_generator(is_multipart: bool):
                 skip_infer_table_types=form_params.skip_infer_table_types,
                 strategy=form_params.strategy,
                 xml_keep_tags=form_params.xml_keep_tags,
-                response_type=media_type,
+                response_type=form_params.output_format,
```

Comment:
The code is simplified now but I think we should limit the possibilities for `form_params.output_format` to a list of acceptable output formats, like it's done with `Literal` here:
https://github.com/Unstructured-IO/unstructured-api/blob/main/prepline_general/api/models/form_params.py#L138

This would require updating the doc as well. Not sure if we need to do anything in `openapi.json` static file then and also consult with `python_client` repo maintainers (cc: @hubert-rutkowski85 )

Except that one - looks ok 👍

Reply:
Thanks for mentioning. I agree about adding Literal to make it more specific and less error prone.

As for python-client, it's defined there as:
```
output_format: Optional[str] = dataclasses.field(default=None, metadata={'multipart_form': { 'field_name': 'output_format' }})
r"""The format of the response. Supported formats are application/json and text/csv. Default: application/json."""
```
It's code generated from openapi.json, where the `output_format` is defined as str. This means that fortunately we don't need to change anything in client now. It's just backend spec that becomes more defined in what it accepts. In the future enums could be used to make openapi.json more specified also on client side, but that's better when done all at once and in separate ticket.

Reply:
Changed the type to `Literal`